datasketches/datasketches.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+#set -x
```

Comment:
This should not be commented out. And the best practice is to use `set -euxo pipefail`